pkg/config/nosystemd.go

```diff
@@ -0,0 +1,10 @@
+// +build !systemd
+
+package config
+
+func getDefaultCgroupManager() string {
+	return "cgroupfs"
+}
+func getDefaultEventsLogger() string {
```

Comment:
Are we missing setting the default to this?

Reply:
"file" is the default